python/src/Arcsine_doc.i.in

```diff
@@ -1,15 +1,16 @@
 %feature("docstring") OT::Arcsine
 "Arcsine distribution.
 
-Available constructors:
-    Arcsine(*a=-1.0, b=1.0*)
```

Comment:
I do not clearly understand your choice here.
- For Arcsine, you removed the "available constructors" section and define the default values of the parameters in the "Parameters" section.
- For WeibullMaxMuSigma, you created a new "Available constructors" section with the default parameters _and_ the "Parameters" section with the default parameters, again.

I am fine with both, but I feel that there is not a single consistent choice in the PR. Am I correct here?

Reply:
I forgot to modify this class. When there is only one constructor, I do not write 'Available constructors'.
I check all the other files to be sure that I did'nt forget other classes....